client/src/components/DataFiles/DataFilesSidebar/DataFilesSidebar.js

```diff
@@ -34,6 +34,8 @@ const DataFilesSidebar = ({ readOnly }) => {
   );
   const { user } = useSelector(state => state.authenticatedUser);
 
+  const sharedWorkspaces = systems.find(e => e.name === 'Shared Workspaces');
```

Comment:
Let's find using the `scheme` instead, in the case that we ever decide to change the name of "Shared Workspaces"
```suggestion
  const sharedWorkspaces = systems.find(s => s.scheme === 'projects');
```